packaging/dashboard.version

```diff
@@ -1 +1 @@
-v2.13.28_
+v2.17.0-beta
```

Comment:
still beta

Reply:
I see [v2.17.0 is released](https://github.com/netdata/dashboard/releases/tag/v2.17.0)

Reply:
Oops! lost in rebase :P